src/views/product-downloads-view/components/downloads-section/downloads-section.module.css

```diff
@@ -5,12 +5,29 @@
 .cardHeader {
   align-items: center;
   display: flex;
+
+  /* Note: logical order is Vault version, then OS.
+     But in flex layout, we want Vault version after OS heading,
+     so that it appears in the top left of the downloads section box */
+  flex-direction: row-reverse;
+  flex-wrap: wrap;
+  gap: 12px;
   justify-content: space-between;
+  padding-bottom: 12px;
+}
+
+.versionSwitcherWrapper {
+  /* Flex-grow to fill space when in separate row */
+  flex-grow: 1;
+  flex-shrink: 1;
+  width: max-content;
 }
 
 .operatingSystemTitle {
   color: var(--token-color-foreground-strong);
-  margin-bottom: 12px;
+
+  /* Flex-grow as much as possible to keep .versionSwitcherWrapper small */
+  flex-grow: 100;
```

Comment:
This is kind of the "magic" part that allows the version-select to wrap and flex-grow to occupy the full row width on mobile... but on mobile, its `flex-grow: 1` is overwhelmed by this `flex-grow: 100` and it ends up at its `max-content` width instead.